tests/fuzz/wasm-mutator-fuzz/workspace/wasm_mutator_fuzz.cc

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) 2019 Intel Corporation. All rights reserved.
```

Comment:
This file is duplicated with `wasm_mutator_fuzz.cc` in parent folder, do we need to keep both files?

Reply:
The one in workspace is for docker building, the one in /wasm-mutator-fuzz is for manually build.

Reply:
Got it, Thanks~